app/main.py

```diff
@@ -16,4 +16,7 @@
 }
 collection_of_coins = {1, 2, 25}
 
-# write your code here
+sorted_variables = {
+    "mutable": [my_favourite_films, marks, collection_of_coins],
+    "immutable": [lucky_number, pi, one_is_a_prime_number, name, profile_info]
+}
```

Comment:
This `sorted_variables` dictionary can be misleading. Even though the concept of mutable and immutable is correct, the way you've applied it isn't. When you put any type of variables into a list (as you have done), the list itself is still mutable, regardless of what it contains. This might lead to confusion. I would suggest to create two separate lists named `mutable_variables` and `immutable_variables` instead of a dictionary. This way, you can avoid confusion and make your code clearer.

Reply:
Як я зрозуміла треба зробити два списка. mutable_variables = [my_favourite_films, marks, collection_of_coins]
immutable_variables = [lucky_number, pi, one_is_a_prime_number, name, profile_info]. Але після цього ламаються мої тести, мені треба їх змінити?

Reply:
As I understood, two lists should be made. mutable_variables = [my_favorite_films, marks, collection_of_coins]
immutable_variables = [lucky_number, pi, one_is_a_prime_number, name, profile_info]. But after that my tests break, do I need to change them?